internal/internal_utils.go

```diff
@@ -469,9 +469,7 @@ func estimateHistorySize(logger *zap.Logger, event *s.HistoryEvent) int {
 			sum += len(event.SignalExternalWorkflowExecutionInitiatedEventAttributes.Input)
 		}
 
-	default:
-		logger.Warn("unknown event type", zap.String("Event Type", event.GetEventType().String()))
-		// Do not fail to be forward compatible with new events
+	default: // Do not fail to be forward compatible with new events
```

Comment:
you can remove default case since it's no-op

Reply:
Done